.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "no-var": off
+    }
+}
```

Comment:
Add a new line to all files

Reply:
Do you mean new line after every single line of code?